src/server/scripts/Northrend/zone_grizzly_hills.cpp

```diff
@@ -375,6 +375,10 @@ class npc_tallhorn_stag : public CreatureScript
                 }
                 _phase = 0;
             }
+
```

Comment:
Already wrote that this commit wasnt intended, shall i open a new pull request or just revert it?